### ISSUE TITLE
TS Error with option "noUnusedLocals"

### ISSUE THREAD

**Issue by SergeShkurko** [drew-y/cliffy#2]
With these tsc option` "noUnusedLocals": true,`
TSC give console errors: 
```
node_modules/cliffy/src/index.ts(4,19): error TS6133: 'Parameter' is declared but its value is never read.
node_modules/cliffy/src/index.ts(10,7): error TS6133: 'columnify' is declared but its value is never read.
```

**Comment by SergeShkurko** [comment]
Resolve this problem Solved the problem by adding types #3

**Comment by drew-y** [comment]
Good catch. I've both merged #3 and removed the unused values from the source code in 4ea52a1998ec.
